src/Murder.Editor/Architect.cs

```diff
@@ -174,18 +174,18 @@ private void QuitToEditor()
         
         internal void PlayGame(bool quickplay, Guid? startingScene = null)
         {
-            startingScene ??= Profile.StartingScene;
+            var actualStartingScene = startingScene ?? Profile.StartingScene;
```

Comment:
can we drop the `var`? 😁 (I should add to .editorconfig...) 
I'm only okay with var when doing super verbose declarations, e.g. `var builder = ImmutableArray.CreateBuilder<int>()`